README.md

```diff
@@ -48,9 +48,7 @@ To install them follow those steps:
 
 1. Download the binary archive from
 
-- Windows: https://www.mediafire.com/file/4lzb21b5tqte8ya/psdk-binaries.7z/file
-- MacOS: https://www.mediafire.com/file/0vnfqx5jkh0adwu/psdk-binaries-macos-arm64.7z/file (Note, this file needs to be updated with new dependencies)
-- Linux: **Not done yet**
+- https://download.psdk.pokemonworkshop.com/binaries/
 
 2. Extract the content of the archive to the root of this project
```

Comment:
Il faut aussi corriger cette ligne car avec ces archives on peut directement extraire dans le dossier psdk-binaries.

Reply:
2. Extract the content of the archive to the psdk-binaries folder. Comme ça ?